backend/controllers/user.js

```diff
@@ -25,30 +25,32 @@ export const updateUser = async (req, res) => {
     const { email_input } = req.params;
     const { name, email, module } = req.body;
 
```

Comment:
It's good that you added some input validators in `validators/user.js`, but you also need to check that the validators passed in your controller. The way you do this is by calling `validationResult(req)` (with `validationResult` from `express-validator`), and if this function returns any errors, then you should return an HTTP 400 Bad Request response. See the Onboarding repo for an example of this: https://github.com/TritonSE/onboarding/blob/main/backend/src/controllers/task.ts#L51C18-L51C34